packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -116,6 +118,12 @@ class FlutterCommandRunner extends CommandRunner<void> {
     argParser.addOption(FlutterGlobalOptions.kPackagesOption,
         hide: !verboseHelp,
         help: 'Path to your "package_config.json" file.');
+    argParser.addFlag(
+      FlutterGlobalOptions.kPrintDtdUri,
+      negatable: false,
+      help: 'Print the URI of the Dart Tooling Daemon, if one is hosted by the Flutter CLI.',
```

Comment:
@christopherfujino is it okay to say "the Flutter CLI" here? Technically it is hosted by DevTools server, but that seemed like an implementation detail that wouldn't mean anything to the end user.